### PR TITLE
BZ1980281: Added bootMode topic to 4.7 version

### PR DESCRIPTION
**Pull request by tmulquee**
Jira ticket: https://bugzilla.redhat.com/show_bug.cgi?id=1980281

For version 4.7 only.

https://deploy-preview-39932--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow#configuring-managed-secure-boot-in-the-install-config-file_ipi-install-configuration-files

### CODE REVIEW

**Comment by dtantsur** [comment]
Looks good but needs rebase

**Comment by tmulquee** [comment]
Noted about the rebase. Thanks Dmitry.

**Comment by tmulquee** [comment]
@dtantsur we have to make the required change a second time, as our original file was deleted during a restructuring of the doc files. Can you please verify the information is present and correct?

**Comment by tmulquee** [comment]
@mjpytlak all edits put through, thanks!

File: modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      address: redfish://<out_of_band_ip> <1>
+      username: <user>
+      password: <password>
+    bootMACAddress: <NIC1_mac_address>
+    rootDeviceHints:
+     deviceName: "/dev/sda"
+    bootMode: UEFISecureBoot <2>
+----
+
+<1> Ensure the `bmc.address` setting uses `redfish`, `redfish-virtualmedia`, or `idrac-virtualmedia` as the protocol. See "BMC addressing for HPE iLO" or "BMC addressing for Dell iDRAC" for additional details.
```
Comment (lpettyjo) [inline modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc:26]: IBM SG. Change the second sentence to have the prepositional phrase start the sentence: "For additional information, see..."

File: modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<2> The `bootMode` setting is `UEFI` by default. Change it to `UEFISecureBoot` to enable managed Secure Boot.
```
Comment (lpettyjo) [inline modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc:28]: IBM SG: Change the second sentence to start with the prepositional phrase: "To enable managed Secure Boot, change..."

File: modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+[NOTE]
+====
+See "Configuring nodes" in the "Prerequisites" to ensure the nodes can support managed Secure Boot. If the nodes do not support managed Secure Boot, see "Configuring nodes for Secure Boot manually" in the "Configuring nodes" section. Configuring Secure Boot manually requires Redfish virtual media.
```
Comment (lpettyjo) [inline modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc:32]: Same comment as above to start the sentence with the prepositional phrase.

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Comment by tmulquee** [comment]
Should be good to go now Lisa.

**Pull request merged**